modules/platform/nuxeo-automation/nuxeo-automation-features/src/main/java/org/nuxeo/ecm/automation/core/operations/services/query/DocumentPaginatedQuery.java

```diff
@@ -70,7 +70,7 @@
     @Param(name = "pageSize", required = false, description = "Entries number per page.")
     protected Integer pageSize;
 
-    @Param(name = "queryParams", required = false, description = "Ordered query parameters.")
+    @Param(name = "queryParams", alias = "searchTerm", required = false, description = "Ordered query parameters.")
```

Comment:
What's the goal of this new alias? Could it be described in the upgrade notes of the ticket?

Reply:
Of course. It's to keep similar arguments to the `Repository.PageProvider` operation ( see https://github.com/nuxeo/nuxeo/blob/master/modules/platform/nuxeo-automation/nuxeo-automation-features/src/main/java/org/nuxeo/ecm/automation/core/operations/services/DocumentPageProviderOperation.java#L76)